model/itemTab.h

```diff
@@ -73,13 +73,13 @@ namespace minsky
     void mouseUp(float x, float y);
     void mouseMove(float x, float y);    
     ItemPtr itemAt(float x, float y);
-    void togglePlotDisplay();
+    void togglePlotDisplay() const;
     void displayDelayedTooltip(float x, float y);        
        
-    ~ParVarSheet() {}
+    ~ItemTab() {}
```

Comment:
destructor should be marked virtual. I think CairoSurface has a virtual destructor, so this isn't a bug yet, but it is safest to always declare a virtual destructor if any other method is declared virtual. I don't know why C++ doesn't make this a feature of the language, maybe there some code out there that relies on non-virtual destructors in polymorphic types.

Reply:
Done, thanks :)